events/events_internal.h

```diff
@@ -27,12 +29,13 @@ void events_freerec(struct eventrec *);
 struct eventrec * events_immediate_get(void);
 
 /**
- * events_network_select(tv):
+ * events_network_select(tv, interrupt_requested):
  * Check for socket readiness events, waiting up to ${tv} time if there are
  * no sockets immediately ready, or indefinitely if ${tv} is NULL.  The value
- * stored in ${tv} may be modified.
+ * stored in ${tv} may be modified.  If ${interrupt_requested} is non-zero
+ * and a signal is sent, exit.
```

Comment:
s/sent/received/

Reply:
I also assume s/is/was

Reply:
hmm, I think "is" makes more sense -- we're saying if a signal arrives *while we're waiting*, right?

Reply:
ok, switched back to "is".  Technically, we'd also exit if the signal arrives while that function is blocked on `poll`, but I'm guessing that's too detailed to mention (and/or can be considered an implementation detail).